layers/+email/mu4e/packages.el

```diff
@@ -39,9 +39,28 @@
                 'ivy-completing-read
               'helm-completing-read))
 
+      (when mu4e-async
+        (require 'smtpmail-async)
+        (mu4e/set-send-function 'async-smtpmail-send-it))
+
+      (when (fboundp 'imagemagick-register-types)
+        (imagemagick-register-types))
+
+      (let ((dir "~/Downloads"))
+        (when (file-directory-p dir)
+          (setq mu4e-attachment-dir dir)))
```

Comment:
Should be in `:init` to allow users to override this settings easily in their `dotspacemacs/user-config`